users/syncing.rst

```diff
@@ -79,6 +79,39 @@ synchronize the file. The block lists are compared to build a list of needed
 blocks, which are then requested from the network or copied locally, as
 described above.
 
+.. _case-sensitivity:
+
+Case Sensitivity in File Names
+------------------------------
+
+In principle, Syncthing works with *case-sensitive* paths, meaning
+that ``file.txt`` and ``FILE.txt`` denote two independent things.
+Consequently, it never considers both as if they were somehow related
+for synchronizing their contents.  Some operating systems
+(e.g. Windows, Mac and Android to an extent) assume the opposite,
+treating them as the same file (or directory) and can never have both
+names simultaneously.
+
+Thus, if a remote device shares a file which would clash with an
+existing local file, it cannot be synchronized to such a system and
+will be reported as a case conflict by Syncthing.  Similarly, if two
+remote devices share differing case variants to your local device,
+only one of them will be pulled, and the other one marked with an
+appropriate error message.  Which one "wins" is currently not
+predictable.
+
+In order to resolve such a case conflict situation, you need to decide
+on a consistent file name and **manually** enforce that across all
+involved, case-insensitive devices.
+
+This cautious behavior tries to save you from possible data loss
+caused by different files overwriting each other's contents.  That
+could have happened before version 1.9.0, where the same file would
+erroneously be accessed under two case-differing file names.
+
+All this does not concern the folder "root" path, but only relative
```

Comment:
```suggestion
All this does not concern the folder path itself, but only relative
```
I would avoid using the word "root", as this whole case-sensitivity question is mostly related to Windows, and the concept of "root" does not exist there, i.e. a typical Windows users does not understand what a "root path" is.

Reply:
Thanks for the review, much appreciated.  There are at least eight uses of the term "root path", "folder root" or similar in the docs already.  I tried to align with that, as I had no better alternative idea.  Will remove the quotes though, for consistency.

The "double space after full stop" is a historic debate with associated scientific research, leaning toward double spaces at least for monospaced fonts. Code and RST markup is usually displayed in such a way.  I mainly go by the [GNU Coding Standards](https://www.gnu.org/prep/standards/standards.html#Comments) and enjoy the benefit of automatic re-wrapping in Emacs.  So yes, on purpose :-)